src/QSwagGenerator/QSwagGenerator.csproj

```diff
@@ -16,7 +16,7 @@
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
-    <Version>1.0.15</Version>
+    <Version>1.1.0</Version>
```

Comment:
Not sure this warranties minor version increase, this is adding what was not considered needed before, not new feature.

Reply:
I understand, thanks Dimitry.
@jondavidford Would it be fine if we put 1.16.0 instead of 1.1.0?

Just let me know.

Reply:
minor version bump would be 1.0.16 I believe. @dmikov what is the difference between adding something not needed before and adding a new feature? :P. But either way doesn't matter to me

Reply:
@jondavidford It's all subjective of course, but if you add ability to annotate or create a doc instead of json, that would be new. People will log the requests for such things as improvement. Adding another http method to list of handled, if requested - people would have added it to the list of issues. It is more of a bug fix or shortcoming fixed.

Reply:
Thanks, both for the explanation.
I will put 1.0.16 as the next version.